python/pyspark/sql/avro/functions.py

```diff
@@ -37,7 +37,7 @@ def from_avro(data, jsonFormatSchema, options=None):
 
     Parameters
     ----------
-    data : :class:`Column` or str
+    data : :class:`pyspark.sql.Column` or str
```

Comment:
This is really just for my info since I don't know docstrings well. What does this part do? is it meant to make 'data' render as a link to the Column class, and it just doesn't work right now?

Reply:
Exactly. It also solves the corresponding warning generated by Sphinx.